neon_llm_core/chatbot.py

```diff
@@ -57,13 +63,12 @@ def ask_chatbot(self, user: str, shout: str, timestamp: str,
         :param timestamp: formatted timestamp of shout
         :param context: message context
         """
-        prompt_id = context.get('prompt_id')
+        prompt_id = context.get('prompt_id') if context else None
```

Comment:
if context is always at least empty dict, we may omit this change

Reply:
The default value is `None`, so we either need this or we could add `context = context or dict()` above.

We could also change the default value from `None` to `{}`, but that does add editor warnings in PyCharm about a mutable default value